client/src/Components/SignIn/SignIn.css

```diff
@@ -0,0 +1,9 @@
+.login-form{
+  width: 50%;
+  max-width: 350%;
```

Comment:
Are we sure that we want the max-width to be over 100%. @MkSona - can you confim that this works with the design?

Reply:
I deleted it.